llmfoundry/data/finetuning/tasks.py

```diff
@@ -307,8 +306,8 @@ def get_preprocessing_fn_from_str(
     def build_from_hf(
         self, cfg: DictConfig, max_seq_len: int,
         tokenizer: PreTrainedTokenizerBase
-    ) -> Union[hf_datasets.DatasetDict, hf_datasets.Dataset,
-               hf_datasets.IterableDatasetDict, hf_datasets.IterableDataset]:
+    ) -> Union[hf_datasets.DatasetDict, hf_datasets.Dataset, hf_datasets.
```

Comment:
I don't think this should include `None` return type. Is the assert below not enough to convince pyright its not None?

Reply:
assert is enough removed